scripts.js

```diff
@@ -0,0 +1,9 @@
+var button = document.getElementsByClassName('button');
```

Comment:
it should be buttons :) because you have more than one button in your html file, also at the name of method that you used to take this elements